plutus-core/executables/uplc/Main.hs

```diff
@@ -322,7 +325,7 @@ runEval (EvalOptions inp ifmt printMode budgetMode traceMode
     let term = void $ prog ^. UPLC.progTerm
         cekparams = case cekModel of
                     -- AST nodes are charged according to the default cost model
-                    Default -> mkMachineParameters semvar PLC.defaultCekCostModel
+                    Default -> PLC.defaultCekParametersForVariant semvar
```

Comment:
And here.

Reply:
I think that one's actually OK: there's a command-line option to tell it which semantic variant to use.